extensions/podman/packages/extension/src/configuration/registry-configuration.ts

```diff
@@ -21,11 +21,13 @@ import { mkdir, readFile, writeFile } from 'node:fs/promises';
 import { homedir, tmpdir } from 'node:os';
 import { dirname, resolve } from 'node:path';
 
-import type { Disposable, QuickPickItem } from '@podman-desktop/api';
-import { commands, env, window } from '@podman-desktop/api';
+import type { DefaultRegistry, DefaultRegistryMirror, Disposable, QuickPickItem } from '@podman-desktop/api';
+import { commands, configuration, env, window } from '@podman-desktop/api';
```

Comment:
question: why do we import configuration where in `image-registry.ts` we inject the `ConfigurationRegistry` (added in the same PR)?